ktlint-test/src/main/kotlin/com/pinterest/ktlint/test/RuleSetProviderTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.pinterest.ktlint.test
+
+import com.pinterest.ktlint.core.RuleSetProvider
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import java.io.File
+
+open class RuleSetProviderTest(
+    private val rulesetClass: Class<out RuleSetProvider>,
+    private val packageName: String
```

Comment:
nit: We could, in theory, get the package name from `rulesetClass` and this would work with current ktlint structure, but in case of custom rulesets, the rules could be located in a different folder, so I would rather leave it like it is now